### ISSUE TITLE
It display error when run the command:

### ISSUE THREAD

**Issue by cendelian** [dbolya/yolact#177]

**Comment by cendelian** [comment]
Dear dbolya,
I am sorry to bother you.

I run the command:
`sh data/scripts/COCO_test.sh`

and after that, I run the command as below:
`python eval.py --trained_model=weights/yolact_base_54_800000.pth`

but it display error as below:
**FileNotFoundError: [Errno 2] No such file or directory: './data/coco/annotations/instances_val2017.json'**

why it display error and why it miss instances_val2017.json?
in other words, what's the steps i did  are wrong?
i just want to run yolact and check it's performance.

could you help me?

**Comment by downhiller** [comment]
I don't suppose you remember what the solution was, @cendelian 🤣
